internal/evaluator/conftest_evaluator.go

```diff
@@ -61,10 +61,7 @@ func NewConftestEvaluator(ctx context.Context, policySources []source.PolicySour
 	}
 
 	dir, err := c.createWorkDir()
```

Comment:
is this `err` no longer being handled?

Reply:
It's part of [Test remove me](https://github.com/hacbs-contract/ec-cli/pull/142/commits/578bdba55d7d46ac3eff378a59102e921dd8f231) commit to test the linter on GitHub Actions

Reply:
> It's part of [Test remove me](https://github.com/hacbs-contract/ec-cli/pull/142/commits/578bdba55d7d46ac3eff378a59102e921dd8f231) commit to test the linter on GitHub Actions

ack. Gotcha!